isolated-contexts.bs

```diff
@@ -10,27 +10,514 @@ Repository: https://github.com/WICG/isolated-web-apps
 Editor: Robbie McElrath 139758, Google LLC https://google.com, rmcelrath@google.com
 
 Abstract:
-  This specification defines "isolated contexts", which allows user agent
+  This specification defines "isolated contexts", which allow user agent
   implementers and specification authors to enable certain features only when
   minimum standards of isolation and integrity are met.
 
 Markup Shorthands: markdown yes
 </pre>
 
 <pre class="link-defaults">
+spec:csp3; type:dfn; for:/; text:csp list
+spec:ecmascript; type:dfn; for:ECMAScript; text:realm
+spec:fetch; type:dfn; for:/; text:response
+spec:fetch; type:dfn; text:fetch params
+spec:fetch; type:dfn; for:fetch params; text:request
+spec:fetch; type:dfn; text:main fetch
+spec:html; type:dfn; for:environment settings object; text:cross-origin isolated capability
+spec:html; type:dfn; text:browsing context group
+spec:html; type:dfn; text:concrete
+spec:html; type:dfn; for:/; text:origin
+spec:html; type:dfn; for:environment settings object; text:origin
+spec:infra; type:dfn; for:list; text:for each
+spec:infra; type:dfn; text:user agent
+spec:url; type:dfn; for:/; text:url
 spec:webidl; type:dfn; text:namespace
 </pre>
+<pre class="anchors">
+urlPrefix: https://w3c.github.io/webappsec-csp/; spec:CSP3
+    type: abstract-op
+        text: Get fetch directive fallback list; url: #directive-fallback-list
+</pre>
+<pre class=biblio>
+{
+  "strict-csp": {
+      "href": "https://web.dev/strict-csp/",
+      "title": "Mitigate cross-site scripting (XSS) with a strict Content Security Policy (CSP)",
+      "authors": [ "Lukas Weichselbaum" ],
+      "date": "15 March 2021"
+  },
+  "securer-contexts": {
+    "authors": [ "Mike West" ],
+    "href": "https://github.com/mikewest/securer-contexts",
+    "title": "Securer Contexts"
+  },
+  "uisecurity": {
```

Comment:
It looks like this manual biblio entry isn't necessary.

Reply:
Yeah, "uisecurity" isn't. How could you tell?